code/modules/client/preference/preferences.dm

```diff
@@ -284,6 +285,8 @@ GLOBAL_LIST_INIT(special_role_times, list( //minimum age (in days) for accounts
 			if(species == "Grey")
 				dat += "<b>Wingdings:</b> Set in disabilities<br>"
 				dat += "<b>Voice Translator:</b> <a href ='?_src_=prefs;preference=speciesprefs;task=input'>[speciesprefs ? "Yes" : "No"]</a><br>"
+			if(species == "Machine")
+				dat += "<b>Synthetic Shell:</b> <a href='?_src_=prefs;preference=ipcloadouts;task=input'>[r_company]</a><br>"
 			dat += "<b>Secondary Language:</b> <a href='?_src_=prefs;preference=language;task=input'>[language]</a><br>"
```

Comment:
I would also like Secondary Language to be above the Grey and Machine species checks. It flows better.